app/src/main/java/org/ole/planet/myplanet/ui/mylife/AdapterMyLife.java

```diff
@@ -85,22 +98,26 @@ public int getItemCount() {
 
     @Override
     public boolean onItemMove(int fromPosition, int toPosition) {
-        RealmMyLife.updateWeight(toPosition+1, myLifeList.get(fromPosition).get_id(),mRealm, myLifeList.get(fromPosition).getUserId());
-        notifyItemMoved(fromPosition,toPosition);
+        RealmMyLife.updateWeight(toPosition + 1, myLifeList.get(fromPosition).get_id(), mRealm, myLifeList.get(fromPosition).getUserId());
+        notifyItemMoved(fromPosition, toPosition);
         return true;
     }
 
     class ViewHolderMyLife extends RecyclerView.ViewHolder implements ItemTouchHelperViewHolder {
         TextView title;
         ImageView imageView;
-        ImageButton editImageButton, dragImageButton;
+        ImageButton editImageButton, dragImageButton, visibility, visibilityOff;
```

Comment:
@ChiragAwale you dont need to create 2 buttons for visibility on and off , just use one button , and change icon when the button is pressed. So the code climate issue will be fixed.

Reply:
Okay